src/components/notifications/noti-tray.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, h, Prop, State, Method } from "@stencil/core";
+
+@Component({
+  tag: "oui-noti-tray",
+  styleUrl: "noti-tray.css"
+})
+export class NotiTray {
+  @State() showNoti = false;
+
+  @Prop({ reflectToAttr: true, mutable: true }) opened: boolean;
```

Comment:
Please add doc comment to public props. This is used by the documentation builder.

```suggestion
  // Drawer status.... blah blah...
  @Prop({ reflectToAttr: true, mutable: true }) opened: boolean;
```